src/Core/AndroidNative/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-all.zip
```

Comment:
Why this downgrade?

Reply:
We're using JDK17...

Reply:
https://docs.gradle.org/current/userguide/compatibility.html suggests 7.3 but I had problems using 7.3 and compiling still so I went slightly higher... When I tried 7.6.x it complained about needing JDK21.